src/utils/utils_common.h

```diff
@@ -40,6 +40,13 @@ typedef enum umf_purge_advise_t {
 #define IS_NOT_ALIGNED(value, align)                                           \
     ((align != 0 && (((value) & ((align)-1)) != 0)))
 #define ALIGN_UP(value, align) (((value) + (align)-1) & ~((align)-1))
```

Comment:
do we still need `ALIGN_UP`? if yes, pls add a comment when to use the safe one

Reply:
Yes, we still need it for cases where `0` is a valid input, like in the `validatePartSize()` function I'm introducing with this PR.